samza-api/src/main/java/org/apache/samza/task/TaskContext.java

```diff
@@ -76,21 +77,21 @@ default Object getUserContext() {
   }
 
   /**
-   * Register a keyed timer with a callback of {@link TimerCallback} in this task.
+   * Register a keyed callback in this task.
```

Comment:
Maybe: "Schedule the {@code callback} for the provided {@code key} to be invoked at provided epoch-time {@code timestamp}"